src/openlcb/BroadcastTime.cxx

```diff
@@ -46,8 +46,10 @@ namespace openlcb
 //
 void BroadcastTime::clear_timezone()
 {
+#ifndef ARDUINO
```

Comment:
Please use `ESP32` instead of  `ARDUINO` since these two APIs are available in a later version of arduino-esp32 (by nature of pulling in ESP-IDF v4.0+).

Reply:
Done.  I sort of guessed as to which preprocessor symbol to use.

Reply:
`ARDUINO` is a good guess, but having it as `ESP32` is better since this library will also work in ESP-IDF env which does not define `ARDUINO`.